### PR TITLE
Slider: onChanged prop logs Keyboard event

### PR DESCRIPTION
**Pull request by maxwellred**
#### Pull request checklist

- [x] Addresses an existing issue: #10292 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Cherry picked functionality from Fabric 7 to Fabric 6

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10370)

### CODE REVIEW

**Comment by aneeshack4** [comment]
@maxwellred By cherrypick do you mean this change is for another branch besides fabric 7? If so, which branch?

**Comment by maxwellred** [comment]
Hi @aneeshack4 Sorry about that I should've clarified in the description but yes this is from Fabric 7 to Fabric 6, thanks!

File: packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx [hunk 1]
```diff
@@ -33,6 +35,7 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
   private _sliderLine = React.createRef<HTMLDivElement>();
   private _thumb = React.createRef<HTMLSpanElement>();
   private _id: string;
+  private _onKeyDownTimer = -1;
```
Comment (aneeshack4) [inline packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx]: I don't see this code in master right now... if it's a cherrypick shouldn't this be in master branch too (7.0)? Or is it implemented differently in 7.0?

Comment (maxwellred) [reply packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx]: Strange, it's in my master. Here's a screenshot:
Screen Shot 2019-09-09 at 6 18 32 PM

Comment (aneeshack4) [reply packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx]: ah my mistake I think I was looking at another file

File: packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md [hunk 2]
```diff
@@ -8129,6 +8129,9 @@ export type OnChangeCallback = (evt?: React.FormEvent<HTMLElement | HTMLInputEle
 // @public (undocumented)
 export type OnFocusCallback = (ev?: React.FocusEvent<HTMLElement | HTMLInputElement>, props?: IChoiceGroupOption) => void | undefined;
 
+// @public (undocumented)
+export const ONKEYDOWN_TIMEOUT_DURATION = 1000;
```
Comment (aneeshack4) [inline packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: Why does this const need to be exported? It wasn't in the original change https://github.com/OfficeDev/office-ui-fabric-react/pull/9530/commits/4c6ff35d07c804286da66afed3421ea2e587f69a

**Pull request merged**